Fix LDFGAS leak to .pc file:
> * QA Notice: pkg-config files with wrong LDFLAGS detected:
> * /usr/lib64/pkgconfig/libbt.pc:Libs: -L${libdir} -lbt -Wl,-O1 -Wl,--as-needed -Wl,--hash-style=gnu -L/usr/lib -L/usr/local/lib 
diff --git a/lib/libbt.pc.in b/lib/libbt.pc.in
index a2a8869..8246cb0 100644
--- a/lib/libbt.pc.in
+++ b/lib/libbt.pc.in
@@ -7,5 +7,5 @@ Name: Blackbox Toolbox
 Description: Utility class library for writing small applications
 Requires: @XFT_PKGCONFIG@
 Version: @VERSION@
-Libs: -L${libdir} -lbt @LDFLAGS@ @ICONV@ @LOCALE@
+Libs: -L${libdir} -lbt @ICONV@ @LOCALE@
 Cflags: -I${includedir}/bt
